### ISSUE TITLE
A11y: Sort buttons should indicate change of state for screen readers

### ISSUE THREAD

**Issue by angelathe** [CDCgov/dibbs-ecr-viewer#1163]
## What needs to be done

Right now, when a user changes the sort direction for a table, screen readers are given no indication about the change. Those using screen readers should be given some sort of announcement when the sort direction changes.

This applies to:
- eCR Library table
- User management table
- Program management table

## Why it needs to be done

This follows the accessibility audit scans of the eCR Viewer.

## Acceptance Criteria

- [ ] Some sort of notification/alert has been added for users using screenreaders when they change the sort direction of a table
- [ ] Manual scans have been run again on these pages/elements, and have come up with no issues.

## Technical Notes

Maybe an aria alert for when the button is enabled? And/or helper text that mentions the filter button will enable after you've made your changes?

## Additional context

https://www.w3.org/WAI/ARIA/apg/patterns/
